package.json

```diff
@@ -1,54 +1,55 @@
 {
-  "name": "freeverse",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/user-event": "^13.2.1",
-    "axios": "^0.24.0",
-    "dayjs": "^1.10.7",
-    "firebase": "^9.6.2",
-    "formik": "^2.2.9",
-    "jest": "^27.4.7",
-    "prettier": "^2.5.1",
-    "prop-types": "^15.8.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-firebase-hooks": "^4.0.2",
-    "react-router": "^6.2.1",
-    "react-router-dom": "^6.2.1",
-    "react-scripts": "5.0.0",
-    "react-testing-library": "^8.0.1",
-    "react-toastify": "^8.1.0",
-    "sass": "^1.47.0",
-    "web-vitals": "^2.1.0"
-  },
-  "scripts": {
-    "postinstall": "$npm_execpath --silent run checkyarn",
-    "checkyarn": "echo \"$npm_execpath\" | grep -q \"yarn\\.js$\" || (echo '⚠️  Use yarn not npm! ⚠️ ' && echo && exit 1)",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "format": "prettier --write \"src/**/*.{js,jsx,scss}\" \"public/*.{html,json}\""
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
-}
\ No newline at end of file
+    "name": "freeverse",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "preinstall": "$npm_execpath --silent run checkyarn",
```

Comment:
Remember to fix that, it wont work on Windows

Reply:
Yup forgot